### PR TITLE
release branch 2.13

### PR DESCRIPTION
**Pull request by berndbischl**
- lqa and elmNN where copied to extralearners, needs to be checked and doced in NEWS

- ranger case weights where handled wrong, reverted. we could update surv ranger.

- travis deploy weird

### CODE REVIEW

File: R/RLearner_surv_ranger.R [hunk 1]
```diff
@@ -11,6 +11,7 @@ makeRLearner.surv.ranger = function() {
       makeIntegerLearnerParam(id = "min.node.size", lower = 1L, default = 3L),
       makeLogicalLearnerParam(id = "replace", default = TRUE),
       makeNumericLearnerParam(id = "sample.fraction", lower = 0L, upper = 1L),
+      makeNumericVectorLearnerParam(id = "case.weights"),
```
Comment (jakob-r) [inline R/RLearner_surv_ranger.R]: :question:

File: R/RLearner_surv_ranger.R [hunk 2]
```diff
@@ -28,7 +29,7 @@ makeRLearner.surv.ranger = function() {
       makeLogicalLearnerParam(id = "keep.inbag", default = FALSE, tunable = FALSE)
     ),
     par.vals = list(num.threads = 1L, verbose = FALSE, respect.unordered.factors = "order"),
-    properties = c("numerics", "factors", "ordered", "featimp", "weights"),
+    properties = c("numerics", "factors", "ordered", "featimp"),
```
Comment (jakob-r) [inline R/RLearner_surv_ranger.R]: :question:

File: R/RLearner_surv_ranger.R [hunk 3]
```diff
@@ -40,7 +41,7 @@ makeRLearner.surv.ranger = function() {
 trainLearner.surv.ranger = function(.learner, .task, .subset, .weights, ...) {
   tn = getTaskTargetNames(.task)
   ranger::ranger(formula = NULL, dependent.variable.name = tn[1L],
-    status.variable.name = tn[2L], data = getTaskData(.task, .subset), case.weights = .weights, ...)
+                 status.variable.name = tn[2L], data = getTaskData(.task, .subset), ...)
```
Comment (jakob-r) [inline R/RLearner_surv_ranger.R]: :question:

File: R/RLearner_classif_ranger.R [hunk 4]
```diff
@@ -7,9 +7,11 @@ makeRLearner.classif.ranger = function() {
     par.set = makeParamSet(
       makeIntegerLearnerParam(id = "num.trees", lower = 1L, default = 500L),
       makeIntegerLearnerParam(id = "mtry", lower = 1L),
+      makeNumericLearnerParam(id = "mtry.perc", lower = 0, upper = 1),
```
Comment (jakob-r) [inline R/RLearner_classif_ranger.R]: :question:

File: R/RLearner_classif_ranger.R [hunk 5]
```diff
@@ -7,9 +7,11 @@ makeRLearner.classif.ranger = function() {
...
       makeIntegerLearnerParam(id = "min.node.size", lower = 1L),
       makeLogicalLearnerParam(id = "replace", default = TRUE),
       makeNumericLearnerParam(id = "sample.fraction", lower = 0L, upper = 1L),
+      makeNumericVectorLearnerParam(id = "case.weights"),
```
Comment (jakob-r) [inline R/RLearner_classif_ranger.R]: :question:

**Comment by pat-s** [comment]
We released this branch to CRAN, so we should merge this into `master`? @berndbischl

**Comment by pat-s** [comment]
merged with #2429, closing

**Pull request closed without merging**